Extensions/autoscroll.js

```diff
@@ -70,41 +76,46 @@ XKit.extensions.autoscroll = new Object({
 			}
 		};
 		$(document).on("click",".auto_scroll_control_step", function(event){
-			// Mark and store the selected speed.
-			controlSteps.removeAttr("selected");
-			$(event.target).attr("selected","true");
-			currentStep = $(event.target).attr("number");
-			XKit.console.add("Scroll index: " + currentStep);
-
-			// Start scrolling
-			var wasScrolling = isScrolling;
-			isScrolling = true;
-			// Function must be started after isScrolling is set
-			// to prevent possible race condition.
-			if(wasScrolling === false)
+			var attr = $(event.target).attr("selected");
+			var ifBool = !$(event.target).is("[selected]");
```

Comment:
I can't say for sure since I wrote the code over 2 years ago (all this changes were from an old commit that the original xkit never implemented) but if I had to guess I tried both because I was having issues.